mslib/msui/flighttrack.py

```diff
@@ -39,8 +39,9 @@
 import os
 
 import fs
-import xml.dom.minidom
-import xml.parsers.expat
+import xml.dom.minidom  # nosec
```

Comment:
I don't like that the nosec marker is on the import, instead of on the actual usage that was deemed unproblematic. This would disable warnings for new places using the module, if any were to be added in the future. Is it possible to instead mark every line using `xml.dom.minidom` as nosec instead, with an explanation why they are safe?

Reply:
explanation moved. At the moment we add a sec scanner we maybe have to configure that differently.